src/markdown/getting-started/create-project.md

```diff
@@ -51,10 +48,21 @@ Once the [Lenra CLI is installed](./install.html), you can use it to create a ne
     [{:.btn.link}Read the specific guide for that](../guides/create-from-scratch.html)
 
 
-## Start **developing**
+## Start developing
 
 Now that you have a new Lenra app project, you can start developing it.
-Be shure you've understand the [Lenra principles](../guides/principles.html) before starting.
+Be shure you've understand the [Lenra principles](./principles.html) before starting since Lenra is a bit different of the classical frameworks.
```

Comment:
```suggestion
Make sure you've understood the [Lenra principles](./principles.html) before starting since Lenra is a bit different from classic frameworks.
```